src/routes/safe/components/Layout/Header/index.tsx

```diff
@@ -34,6 +43,7 @@ const LayoutHeader = (props) => {
           <Row>
             <Heading className={classes.nameText} color="primary" tag="h2" testId={SAFE_VIEW_NAME_HEADING_TEST_ID}>
               {name}
+              <strong>{totalBalancesFormatted ? ` | ${totalBalancesFormatted} ${currentCurrency}` : ''}</strong>
```

Comment:
why would we render an empty `<strong>` tag? or add it to the DOM anyway?

thinking on something around
```tsx
{name}
{!!totalBalancesFormatted && (
  <strong>
    | {totalBalancesFormatted} {currentCurrency}
  </strong>
)}
```

Reply:
Yep! great idea, it's done